.github/workflows/vpat.yml

```diff
@@ -0,0 +1,49 @@
+name: VPAT
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * SUN' # At 00:00 on Sunday
```

Comment:
@dequelabs/cauldron How often do we want the VPAT to be generated? On another project, I had set this to run on a PR merge to `main`.

https://github.com/dequelabs/a11y-metrics-poc#when-to-run-the-vpat-workflow

Reply:
I think merging to `master` is fine, since we would be releasing changes that might effect the vpat.

Reply:
Cool. I can setup the workflow to merges to `master`.

Reply:
I might take over this soon, unless you wanted to bring things to completion. @isner made some changes to support custom labels, although I need to sit down and figure out they need to be in cauldron.

Reply:
Is there anything else to do besides updating the workflow?

Reply:
Mostly just clarifying the labels so it's clear which ones are used for vpats and others are for categorization.